app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -226,7 +226,12 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.metrics_verify != '' && $scope.angularForm.metrics_verify.$valid)) {
       return true;
     } else if(($scope.currentTab == "default" &&
-      ($scope.emsCommonModel.ems_controller == "ems_container" || $scope.emsCommonModel.ems_controller == "ems_middleware")) &&
+      (
+         $scope.emsCommonModel.ems_controller == "ems_container" ||
+         $scope.emsCommonModel.ems_controller == "ems_middleware" ||
+         $scope.emsCommonModel.ems_controller == "ems_datawarehouse"
+      )
```

Comment:
Maybe this would be the time to change this to something like `[ "ems_container", "ems_middleware", "ems_datawarehouse" ].includes($scope.emsCommonModel.ems_controller)` ?